lib/presentation/utils/ui_utils.dart

```diff
@@ -1,5 +1,19 @@
 import 'package:flutter/material.dart';
 import 'package:fluttertoast/fluttertoast.dart';
+import 'package:intl/intl.dart';
+
+/// gives the current screen width
+double getScreenWidth(BuildContext context) =>
+    MediaQuery.of(context).size.width;
+
+/// gives the current screen height
+double getScreenHeight(BuildContext context) =>
+    MediaQuery.of(context).size.height;
+
+String getTodoCardDisplayDate(DateTime date) {
+  final DateFormat formatter = DateFormat('MMMM dd');
```

Comment:
@dilumdesilva we should avoid hard codings. This 'MMMM dd' can be put inside constants and used from there.

Reply:
@saurabhgodbole in ui_utils I see other things like font size is hardcoded by the way we need to move them as well.

Reply:
@dilumdesilva you mean toast methods, yes we need to use font manager constant there.